sample/res/drawable/drawer_shadow.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:angle="0"
+        android:startColor="#55000000"
+        android:endColor="#00000000"/>
```

Comment:
You have a colors.xml file you should use to put this values there. Please, move this hardcoded values to colors.xml